.esbuild/build.ts

```diff
@@ -56,7 +56,9 @@ const handler = (e) => {
 
 const main = async () => {
   await generateLangium();
-  await mkdir('stats').catch(() => {});
+  await mkdir('stats').catch(() => {
+    // Ignore error if directory already exists
+  });
```

Comment:
```suggestion
  await mkdir('stats', {recursive: true});
```

Setting `options.recursive = true` will also ignore errors if the directory already exists, see https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fspromisesmkdirpath-options